lib/src/mock_storage_reference.dart

```diff
@@ -59,5 +59,5 @@ class MockStorageReference extends Mock implements StorageReference {
   FirebaseStorage getStorage() => _storage;
```

Comment:
This getter and the others (bucket, path, name) have all been replaced, so you'll have to update those too.

You can refer to the Dart Analyzer warnings to find out which ones you need to update:

![image](https://user-images.githubusercontent.com/3399854/111015071-5f562880-83ea-11eb-9b12-de5d91c9048e.png)

Doc:
![image](https://user-images.githubusercontent.com/3399854/111015044-3e8dd300-83ea-11eb-9c23-c9513f1a423e.png)